app/models/annual_produce.rb

```diff
@@ -1,6 +1,5 @@
 class AnnualProduce < Produce
-  has_many :Crops
-  validates :name, presence: true, format: { with: /\A[a-zA-Z]*\Z/ } # check if the name has only letters
-  validates :name, :uniqueness => { :case_sensitive => false } # only one produce of a particular name should exist
-  validates :duration, presence: true, format: { with: /\A[0-9]*\Z/ } # duration should only be numbers
+  validates :duration, presence: true
+  validates_absence_of :start_date
+  validates_absence_of :end_date
 end
```

Comment:
Just want to make sure that I merged this correctly; should the earlier validations still be in place?

Reply:
This should be fine too as AnnualProduce inherits the validations from Produce.